fiona/drvsupport.py

```diff
@@ -141,6 +141,23 @@
 ])
 
 
+# Mininmal gdal version for different modes
+driver_mode_mingdal = {
+
+    'r': {'GPKG': (1, 11, 0)},
+
+    'w': {'GPKG': (1, 11, 0),
+          'PCIDSK': (2, 0, 0)},
+
+    'a': {'GMT': (2, 0, 0),
+          'GPKG': (1, 11, 0),
+          'GeoJSON': (2, 1, 0),
+          'GeoJSONSeq': (2, 0, 0),
```

Comment:
@rbuffat I am not sure that the GeoJSONSeq driver actually supports appending. There was a brief exchange about this on the gdal-dev list a few weeks ago. Also I think the min version might be 2.3.

Reply:
yes, I forget to remove this. Before I changed GeoJSONSeq to test for the original driver it missed that it defaulted to the GeoJSON driver. I added a test to ensure the modes of driver_mode_mingdal are also present in supported_drivers.